### DISCUSSION TITLE
How to stream m3u8 to youtube using Ant-Media?

### DISCUSSION THREAD

**Discussion by MrNate592** [ant-media/Ant-Media-Server#5690] [Q&A]
Hello I am running Ant Media Community edition and I would like to stream using a m3u8 stream url to broadcast to youtube, is it possible to do so?

**Comment by tremzo** [comment]
@MrNate592 Ant Media Server cannot ingest m3u8 directly to broadcast to YouTube as an RTMP endpoint. You'd need to convert the m3u8 into another format first that is compatible with AMS for instance, RTMP, or SRT. 

Is the m3u8 from another live stream or do you have all the HLS files saved?

**Reply by MrNate592** [reply]
It's from another live stream

**Comment by dmtan90** [comment]
Hi @MrNate592. You can add a new Stream Source and paste your m3u8 link as Stream URL in the form. After that, you can add your Youtube endpoint in Edit RTMP Endpoints.

**Reply by MrNate592** [reply]
Thanks but the stream is not sending data to youtube or when I try to view it, it's just stuck on loading. Here is an example m3u8 I'm trying to stream. https://demo.unified-streaming.com/k8s/features/stable/video/tears-of-steel/tears-of-steel.ism/.m3u8

**Reply by lastpeony** [reply]
Hi,
How do you publish to AMS?
Just add youtube RTMP point to the live stream.
This is not related to hls

**Reply by dmtan90** [reply]
@MrNate592 you can update "streamFetcherBufferTime=3000" in AppSettings and re-check again
![image](https://github.com/ant-media/Ant-Media-Server/assets/12541784/09064a66-7519-43a7-a3b6-e9502d90c26f)
